### PR TITLE
Updated TextInput multiline property

### PR DESCRIPTION
**Pull request by mnaderian**
Updated the TextInput multiline property's docs. Part of the text needs to be styled as a note.

### CODE REVIEW

File: docs/textinput.md [hunk 1]
```diff
@@ -437,7 +437,9 @@ Limits the maximum number of characters that can be entered. Use this instead of
 
 ### `multiline`
 
-If `true`, the text input can be multiple lines. The default value is `false`. It is important to note that this aligns the text to the top on iOS, and centers it on Android. Use with `textAlignVertical` set to `top` for the same behavior in both platforms.
+If `true`, the text input can be multiple lines. The default value is `false`. 
+
+> It is important to note that this aligns the text to the top on iOS, and centers it on Android. Use with `textAlignVertical` set to `top` for the same behavior in both platforms.
```
Comment (Simek) [inline docs/textinput.md:442]: This is the old notice formatting which will be replacing successfully with Docusuaurs admonitions. 

Please use `info` or `note` one instead  the blockquote in there:
* https://docusaurus.io/docs/markdown-features/admonitions

**Review by Simek — CHANGES_REQUESTED** [review]
Hello @mnaderian, thank you for the update! 👍 

I have left a suggestion, please address that, correct lint and PR should be ready to ship!

File: docs/textinput.md (hunk 1 above)
Comment (mnaderian) [reply docs/textinput.md]: Update is done. hope I haven't screw anything up! 😄

Comment (Simek) [reply docs/textinput.md]: Looks good, however lint still complains about unnecessary space in line 440. Please check is there is no trailing spaces, or if the empty line aster the mentioned line did not contains space.

File: docs/textinput.md [hunk 2]
```diff
@@ -437,7 +437,11 @@ Limits the maximum number of characters that can be entered. Use this instead of
 
 ### `multiline`
 
-If `true`, the text input can be multiple lines. The default value is `false`. It is important to note that this aligns the text to the top on iOS, and centers it on Android. Use with `textAlignVertical` set to `top` for the same behavior in both platforms.
+If `true`, the text input can be multiple lines. The default value is `false`.
```
Comment (Simek) [inline docs/textinput.md:440]:
```suggestion
If `true`, the text input can be multiple lines. The default value is `false`.
```

I think that this space annoy lint, however I was not able to directly commit to your branch to confirm that.

**Review by Simek — APPROVED** [review]
LGTM, thank you for the update @mnaderian! 👍

**Pull request merged**